examples/kitchensink/app-counter/test/config/bootstrap.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BehaviorSubject } from 'rxjs';
+import { jsdom } from 'jsdom';
+
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import { mount } from 'enzyme';
+
+global.React = React;
+global.ReactDOM = ReactDOM;
+global.BehaviorSubject = BehaviorSubject;
```

Comment:
For the test yes, if you want I can remove it and perform all the includes in the file `Root.spec.js`